package.json

```diff
@@ -217,11 +218,11 @@
     "envify": "^4.1.0",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.15.1",
-    "eslint": "^6.8.0",
-    "eslint-plugin-babel": "^5.3.0",
+    "eslint": "^7.7.0",
     "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-mocha": "^6.3.0",
-    "eslint-plugin-react": "^7.18.3",
+    "eslint-plugin-mocha": "^8.0.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-react": "~7.20.0",
```

Comment:
I updated this to use a `~` range because on `v7.21.x` I kept seeing false positives for the `react/no-unused-prop-types`, in cases where we use `PropTypes.arrayOf(PropTypes.shape({ ... }))` specifically.

I haven't found a similar bug report for this yet, so I'm going to do a little more digging and create one if necessary. In the meantime, I think `~7.20.0` has everything we need.